frontend/src/utils/python-poet/poet.ts

```diff
@@ -33,15 +33,69 @@ export class Variable implements PythonCode {
 }
 
 export class Literal implements PythonCode {
-  constructor(public value: unknown) {}
+  constructor(
+    public value: unknown,
+    public objectAsFieldNames = false,
+    public removeNull = false,
+    public removeUndefined = true,
```

Comment:
we should change this to:

```
    public value: unknown,
    public opts: {objectAsFieldNames?: boolean, removeNull?: boolean, removeUndefined?: boolean}
```

it improves readability and otherwise it will be very easy to mix them up